dymos/examples/ssto/doc/test_doc_ssto_earth.py

```diff
@@ -45,18 +43,18 @@ def test_doc_ssto_earth(self):
         phase.set_time_options(initial_bounds=(0, 0), duration_bounds=(10, 500))
 
         phase.add_state('x', fix_initial=True, ref=1.0E5, defect_ref=1.0,
-                        rate_source='eom.xdot', units='m')
+                        rate_source='xdot', units='m')
```

Comment:
targets and units here are no longer necessary, since the targets have the same name as the states/controls/parameters.